src/Microsoft.Azure.WebJobs/IBinder.cs

```diff
@@ -1,26 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Microsoft.Azure.WebJobs
-{
-    /// <summary>
-    /// Interface defining functionality for dynamically binding to WebJobs SDK attributes
-    /// at runtime.
-    /// </summary>
-    /// <remarks>This interface enables imperative binding with attribute information specified at runtime.</remarks>
-    public interface IBinder
```

Comment:
Yeah - I also realized that it would be breaking for Functions, though we've encouraged people to use Binder there. I'll rework this PR to just add the new overload I was trying to add in the first place w/o removing this. We might also consider marking it as obsolete so people have time to merge over.